### PR TITLE
Added company name and fork icon to facilitate repo identification.

### LINKED ISSUE #105: As a user I should be able to see the organization / user owned by a repo
This would help add distinction between upstream and forks.  It could simply be prefixing the repo name with the organization, eg. _some-organization/this-repo_

### PR DESCRIPTION
**Pull request by jpoo90**
Closes #105 

* Added organization name.
* Added fork icon, to identify forks.
* Updated tests.

screen shot 2017-04-27 at 1 52 37 pm

### CODE REVIEW

**Comment by thescientist13** [comment]
Can one of the admins verify this patch?

File: src/components/repositories-table/repositories-table.tsx [hunk 1]
```diff
@@ -33,6 +33,12 @@ class RepositoriesTable extends React.Component<RepositoriesTablePropsInterface,
     return filter !== '' && repositoryName.indexOf(filter) < 0;
   }
 
+  isFork(fork: boolean): any {
+    if(fork) {
+      return <div className="fork-icon"></div>;
```
Comment (thescientist13) [inline src/components/repositories-table/repositories-table.tsx]: @jpoo90 
my thoughts here are "mixed" if you will, and can pardon a soon to be pun.

Current convention has just been more or less just HTML with a sprinkle of logic.  Here we introduce a mix of templating and JS, if you will
```javascript
return <tr key={index} className={repository.hasAssignedIssues ? 'bg-danger' : ''}>
              <td>
                <a target="_blank" href={repository.url}>{(index + 1) + ') ' + repository.fullName</a>
                {this.isFork(repository.isFork)}
              </td>
              <td>{repository.issues? repository.issues.length : 0} </td>
```

Should we put more template generation into the class, or just add a conditional class name to an element, whether it is a fork or not?

Curious what the recommended approach is here, as I am not 100% sure and can see value in both approaches.  Would just favor a general strategy overall.

Thoughts?

Comment (jpoo90) [inline src/components/repositories-table/repositories-table.tsx]: @thescientist13 

I think in this case it's more about a preference. [React documentation](https://facebook.github.io/react/docs/conditional-rendering.html) has both methods as valid approaches.  I prefer having functions because I think the code looks cleaner. 

Another option can be extracting that logic into a `RepoNameComponent` which would handle the logic internally to return the final component.

Comment (thescientist13) [inline src/components/repositories-table/repositories-table.tsx]: seems like a matter of defining a proper connvetion / styleguide for the project, which seems out of scope for this particular task though.

**Comment by thescientist13** [comment]
ok to test

**Comment by thescientist13** [comment]
hmmm... I wonder why test results weren't generated.  reviewing now.

**Comment by thescientist13** [comment]
@jpoo90 
I found the issue, higher up in the output.  Looks like it's just a simple linting error
```
12:48:06 yarn run v0.21.3
12:48:07 $ npm run clean && export TEST_REPORT_PATH=./reports/test-results && eslint ./src/**/**/*.spec.jsx && jest --coverage --config ./jest.config.json --coverage
12:48:07 
12:48:07 > github-dashboard@1.1.0 clean /var/lib/jenkins/workspace/CI-github-dashboard
12:48:07 > rimraf ./build && rimraf ./reports && mkdir ./reports && mkdir ./reports/test-results
12:48:07 
12:48:08 
12:48:08 /var/lib/jenkins/workspace/CI-github-dashboard/src/components/repositories-table/repositories-table.spec.jsx
12:48:08   178:6  error  A space is required after ','  comma-spacing
12:48:08 
12:48:08 ✖ 1 problem (1 error, 0 warnings)
12:48:08 
```
